mkosi/__init__.py

```diff
@@ -2128,6 +2128,9 @@ def install_uki(
             boot_binary,
         )
 
+        if ArtifactOutput.pcrs in context.config.split_artifacts and pcrs:
+            (context.staging / context.config.output_split_pcrs).write_text(json.dumps(pcrs))
```

Comment:
This doesn't work if there's more than one UKI in the image, we'll just overwrite the pcrs file again and again (and we loop over kernels from highest to lowest version). We should handle this in `install_kernel()` and only keep the pcrs file for the UKI with the highest version (the first one) as that is the one we extract as a split artifact.

Reply:
If there are multiple ones and only one gets signed it's still problematic though - is this something that happens in practice?

Reply:
I'll start with processing the first one as suggested, and added a TODO to consider the case of multiple ukis in the future